src/Components/Forecast/BookmarkIcon/bookmarkIconComponent.jsx

```diff
@@ -0,0 +1,15 @@
+import './bookmarkIconStyles.css'
+import {BsBookmarkHeartFill} from 'react-icons/bs'
+import { useState } from 'react'
+
+const BookmarkIcon= ()=>{
+const [isClicked,setIsClicked] =useState(false)
+
+return(
+     <BsBookmarkHeartFill className="bookmark-icon" style={{color:isClicked?"rgb(17, 64, 110)":"#333"}} onClick={()=>setIsClicked(!isClicked)}/>
```

Comment:
icon doesn't change on click

Reply:
I'll take a look at it ...for now its supposed to change its color when clicked